### ISSUE TITLE
StaxRip 2.13.0 and MPEG2source error over remote drive

### ISSUE THREAD

**Issue by jlw4049** [staxrip/staxrip#929]
StaxRip throws a D2V error saying **couldn't open d2v file is it read only** when trying to encode over the network on a network drive via **Batch** mode

I had to map the drive inside windows

Example
`\\JLWSERVER\file.mpg` does not work
`X:\file.mpg` DOES work and will not throw the above error

I'm not sure if this error is exclusive to dgdecode or how StaxRip accesses network drives

**Edit: Error is exclusive to dgdecode and batch over remote drives** 

**Edit:** Further errors
```
   at StaxRip.Proc.Start() in D:\Code\Github\staxrip\staxrip\Source\General\Proc.vb:line 361
   at StaxRip.CommandLineDemuxer.Run(Project proj) in D:\Code\Github\staxrip\staxrip\Source\General\Demux.vb:line 152
   at StaxRip.MainForm.Demux() in D:\Code\Github\staxrip\staxrip\Source\Forms\MainForm.vb:line 3412
   at StaxRip.MainForm.OpenVideoSourceFiles(IEnumerable`1 files, Boolean isEncoding) in D:\Code\Github\staxrip\staxrip\Source\Forms\MainForm.vb:line 2215
```

**Comment by Discograp** [comment]
this video file Bugs! from the Zyber? if yes, then it uses bad software to insert subtitle at the end of the video

it breaks the encoding of the file

**Comment by Dendraspis** [comment]
As far as I remember this should be related to a wrong DLL, that was included in v2.13.
Please check v2.14, the problem should be fixed.

**Comment by jlw4049** [comment]
You are correct and it's solved thanks
